packages/extension/src/background/transactions/transactionMessaging.ts

```diff
@@ -499,6 +499,44 @@ export const handleTransactionMessage: HandleMessage<
         })
       }
     }
+
+    case "REMOVE_MULTISIG_OWNER": {
+      try {
+        const { address, signerToRemove, newThreshold } = msg.data
+
+        const signersPayload = {
+          entrypoint: "removeSigners",
+          calldata: stark.compileCalldata({
+            new_threshold: newThreshold.toString(),
+            signers_to_remove_len: "1",
+            signers_to_remove: utils.hexlify(
+              utils.base58.decode(signerToRemove),
+            ),
+          }),
```

Comment:
```suggestion
          calldata: stark.compileCalldata({
            new_threshold: newThreshold.toString(),
            signers_to_remove: utils.hexlify(
              utils.base58.decode(signerToRemove),
            ),
          }),
```

Don't need to add `*_len`